optimize_sidebars.js

```diff
@@ -1,4 +1,4 @@
-function docsComponentsLink(label, href) {
+function docsLink(label, href) {
```

Comment:
renamed this helper because it was too specific before. Many other changes in this file are a result of the rename.